TESTING.md

```diff
@@ -0,0 +1,99 @@
+# Test Strategy
+
+The overall test strategy can be described as *minimalistic*, rather than extensive. The goal of the tests is to verify that the application is deployed properly, so we will verify the main application features. 
```

Comment:
```suggestion
The overall test strategy can be described as _minimalistic_, rather than extensive. The goal of the tests is to verify that the application is deployed properly, so we will verify the main application features. 
```